<?xml version="1.0"?>
<launch>
    <include file="$(find diff_drive_base1)/launch/description.launch"/>
    <include file="$(find diff_drive_base1)/launch/rosserial.launch"/>

    <rosparam command="load" file="$(find diff_drive_base1)/config/controllers.yaml"/>

    <node name="robot_node" pkg="diff_drive_base1" type="diff_drive_base_node1" output="screen">
        <param name="max_speed" value="1.0"/>
    </node>

    <node name="base_controller_spawner" pkg="controller_manager" type="spawner" args="
        robot_joint_publisher 
        robot_velocity_controller"/>
    <node name="robot_state_publisher" pkg="robot_state_publisher" type="robot_state_publisher"/>
</launch>